connectors/sources/s3.py

```diff
@@ -311,11 +315,33 @@ def get_default_configuration(cls):
                 "type": "list",
                 "value": "ent-search-ingest-dev",
             },
+            "aws_access_key_id": {
+                "display": "textarea",
+                "label": "AWS Access Key Id",
+                "order": 2,
+                "type": "str",
+                "value": "A1B2C3D4",
+            },
+            "aws_secret_access_key": {
+                "display": "textarea",
+                "label": "AWS Secret Key",
+                "order": 3,
+                "type": "str",
+                "value": "A1B2C3D4",
+                "sensitive": True,
+            },
+            "region": {
+                "display": "textarea",
+                "label": "AWS Region name",
+                "order": 4,
+                "type": "str",
+                "value": "us-east-1",
+            },
```

Comment:
How does it affect the S3 connector?

Does it restrict which buckets can be synced?

Reply:
No there is no restrictions with this but as we discussed on call previously that if we are not passing region then we get access denied error.. That's why we need to pass it as of now.. 
We are trying to take update on this:https://github.com/aio-libs/aiobotocore/issues/1005#issuecomment-1530383313
once we get the update on this so we can check confirm on this.

Reply:
We were facing some exception related to None type region when try to run the connector without passing region name. However it's not reproducible now and looks like there might be some issue in our s3 configuration that time. 

We tested our connector removing the region as you suggested and found that without passing region name our connector is working fine with all type of buckets. 
So we can remove the dependency of region & I've updated PR with the same too